contribute.html

```diff
@@ -0,0 +1,434 @@
+								<h3>Give help / Answer questions</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> There are many questions on GitHub, and also on Forum. Maybe for some of the issues, we need more insights.</p>
+							<p>Sometimes there are open questions where you could know the answer. 
+								<br/>Or help is needed for a specific installation (e.g. a driver installation for a specific SBC, or a check of a performance tweak etc.)
+								<br/><br/>Browse <a href="https://github.com/MichaIng/DietPi/issues" target="_blank" rel="noopener">DietPi issues on GitHub</a> and find these items.
+							</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 3 -->
+				<!-- Start details for portfolio project 4 -->
+				<div id="slidingdiv4" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/test-configurations.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/test-configurations.jpg" alt="GNUstep" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Test configurations</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Very similar with investigate issues, but more specific to hardware - either specific SBC models or PCs</p>
+							<p>There is often a need to check a new installation on a specific SBC (Single Board Computer), or maybe how DietPi runs with a specific driver. This time to do such installation provides a big help to the overall project.</p>
+							<p>Examples:
+								<ul>
+									<li>A driver installation for a specific SBC</li>
+									<li>or check of a performance tweak.</li>
+								</ul>
+								Browse <a href="https://github.com/MichaIng/DietPi/issues?q=is:open+is:issue+label:%22Help+wanted+:sos:%22+label:%22Testing/testers+required+:arrow_down_small:%22"><i>Help wanted !</i> or <i>Testing Required</i></a> in DietPi issues on GitHub.</p>
+							</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 4 -->
+				<!-- Start details for portfolio project 5 -->
+				<div id="slidingdiv5" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/improve-website.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/improve-website.jpg" alt="Chromium" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Improve DietPi website</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> On the website it's always something to do. Someone with skills in graphics / web design could provide help.</p>
+							<p>Have you seen something on the website which could be improved ? Do you have web design skills and you would like to help ?</p>
```

Comment:
No blank before question marks. 2x.

e.g. "... improved ?" -> "improved?"

Reply:
Probably a style/visual question as well, but let's stick with correct notation and skip the space between words and punctuation in all cases.